app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="app_name">multimatum</string>
+    <string name="log_in">Log in</string>
```

Comment:
Not sure about the name you use, maybe something more explicit like "log_in_text" would be better

Reply:
sounds fair